### DISCUSSION TITLE
MA Database reset after dismounting a share

### DISCUSSION THREAD

**Discussion by rvandet** [music-assistant/support#1304] [Q&A] [answered]
When i dismount my network drive (Samba share), the database still has a lot of Albumartists and Artists Tags, that belonged to the MP3 from this network drive.

Deleting each Album and Artist is tediuos and time consuming. Is there a quick way?

**Comment by marcelveldt** [comment, marked as answer]
This is actually an open bug. If you delete a provider entry from the settings it should cleanup but this seems to be failing.
For now easy way is to simple remove the addon and reinstall it or if you have access to the data files, delete the library.db and cache.db files

**Comment by rvandet** [comment]
Which folder is the library.db and cache.db

**Reply by TomHilde** [reply]
Do you know by now where the two files are stored?

**Comment by rvandet** [comment]
My guess is that the db was growing. The ssd use was 55%. After removing and reinstalling MA, i have 25% ssd use.

**Comment by rvandet** [comment]
Since a few updates ago this was solved.

**Reply by marcelveldt** [reply]
Correct, this is solved a few days ago and provider entries are properly cleaned up after removal.
